.eslintrc

```diff
@@ -78,6 +78,6 @@
     "comma-spacing": ["error", { "before": false, "after": true }],
     "eol-last": "error",
 
-    "prettier/prettier": "error"
+    // "prettier/prettier": "error"
```

Comment:
temp will remove. used to VSD code's format-on-save but can't seem to get it to play nice with the linter files.